libs/minio-operator/config.jsonnet

```diff
@@ -1,22 +1,23 @@
 local config = import 'jsonnet/config.jsonnet';
 local versions = [
-   'v5.0.15',
-   'v5.0.14',
-   'v5.0.13',
-   'v5.0.12',
-   'v5.0.11'
+  'v6.0.0',
+  'v5.0.15',
+  'v5.0.14',
+  'v5.0.13',
+  'v5.0.12',
+  'v5.0.11'
```

Comment:
I'm wondering: Since these are minor version changes, can we consolidate them to one? This will help decreasing the amount of CI time